projects/plugins/jetpack/extensions/shared/external-media/media-button/media-menu.js

```diff
@@ -11,12 +11,11 @@ import { media } from '@wordpress/icons';
 import MediaSources from './media-sources';
 
 function MediaButtonMenu( props ) {
-	const { mediaProps, open, setSelectedSource, isFeatured, isReplace } = props;
+	const { mediaProps, open, setSelectedSource, isFeatured, isReplace, hasImage } = props;
 	const originalComponent = mediaProps.render;
-
-	if ( isFeatured && mediaProps.value === undefined ) {
-		return originalComponent( { open } );
-	}
+	let isPrimary = isFeatured;
+	let isTertiary = ! isFeatured;
+	const extraProps = {};
```

Comment:
Nitpick: maybe this should just be `let isSecondary = false;`? I'm not sure I see an advantage to using an object here (unless all the `is*` props here are added to it)

Reply:
I enjoy nitpicks, and I would overall prefer that, too.

My reasoning is that I was trying to keep the surface area of changes as limited as possible, given the broad and varied use of media buttons throughout different parts of the Gutenberg UI. I wasn't confident that always adding the `isSecondary` prop (even set to `false` in all non-featured image uses) wouldn't disrupt those other uses.

Now, with a bit more distance, I can see that it will be fine. The prop is `false` by default in the Button component and will be in all situations save the "Replace Image" use here. I'll update it that way.